ResearchJournal.tex

```diff
@@ -0,0 +1,101 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage{url}
+\usepackage[autostyle]{csquotes}
+
+% Please include a clear, concise, and descriptive title
+\title
+{Research Journal}
+
+% Please do not change the subtitle
+\subtitle{COMP130}
+
+% Please put your student number in the author field
+\author{Student number: 1607539}
+
+\begin{document}
+
+\maketitle
+
+\blockquote {A journey of a thousand miles begins with a single step - Lao Tzu}\\
+\indent\blockquote {On second thoughts, let's not go to Camelot. 'Tis a silly place - Graham Chapman}%{All the roads we have to walk are winding - Noel Gallagher}{I got the right street, but the wrong direction - Rufus, Chaka Khan}
+
+\section*{\underline{Pathfinding in games}}
+
+Pathfinding stands out as one of the most common applications of AI research to games \cite{botea2013pathfinding}.  In practice the process must be as efficient as possible as there many are many other demands on the CPU, such as physics and graphics \cite{sturtevant2007memory}.  For the optimum solution, A* is known as one of the best search algorithms \cite{algfoor2015comprehensive}.
+
+\subsection*{A*}
+
+By extending Dijkstra's algorithm to use heuristics relating to an end node (in Dijkstra's algorithm H is always 0) \cite{lester2005pathfinding}, A* is the \textit{de facto} pathfinding algorithm \cite{leigh2007using}, and with an admissible heuristic, returns optimal solutions.  Dijkstra's algorithm was the only choice for pathfinding until the last two decades \cite{cui2011direction}, and should still be used when there is no end node, for example, were there several targets and an actor were needed to move to the closest \cite{lester2005pathfinding}.
+
+\subsection*{Heuristics}
+
+The Oxford dictionary defines Heuristics in regards to computing as proceeding to a solution by trial and error or by rules that are only loosely defined.  A* uses a weighting for each tile defined as the sum of the distance from the start node, and the distance from the target as a heuristic (H), to find the path of least cost. Commonly written as finding the smallest value of $f(n) = g(n) + h(n)$.  
```

Comment:
Is the OED definition a direct quotation -- if so, make it clearer. Either way, OED should be cited in your bibliography.

However beware of using dictionary definitions, as many words mean something different in everyday usage than in technical contexts. Here the definition of "heuristic" doesn't quite capture its usage in A* search for example (as a way for the search to prioritise the most promising out of several options).